microsoft-edge/web-platform/web-platform.md

```diff
@@ -5,12 +5,24 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
-ms.date: 10/06/2024
+ms.date: 02/04/2025
 ---
 # Development tips for Microsoft Edge
 
+<!-- compare https://developer.chrome.com/docs/web-platform 
+"Web Platform
+Read the documentation for Web Platform APIs, some of which are experimental in Chrome.  For example, origin and developer trials, or features only found in Canary." -->
+
+<!-- first sentence explains and disambiguates/clarifies the h1 title & the TOC bucket name: -->
+This TOC bucket is named Web Platform and its scope is Edge-specific web-platform considerations.<!-- todo: elim "toc bucket" -->
```

Comment:
get rid of "toc bucket" wording; clean up top of page

Reply:
your lead-in says "sections:" but below that you propose to link to all "articles"

Reply:
no need for list of h2 sections; already present in "In this article" nav.  deleting such list.

Reply:
I question "It would be good to list all articles in a bullet list right below this sentence."
"In this article" nav has a list of all h2 sections.  In this type of bucket overview/nav page, there's an h2 heading/section for each article that's in the TOC bucket, w/ a link to that article.
@captainbrosset 
Closing this comment b/c used the improved intro wording describing this toc bucket scope.